doc/general/operation-modes/operation-modes.md

```diff
@@ -34,6 +34,38 @@ The queue nutshell serves to encapsulate the functionality of a receipt chain fo
 
 The Helper nutshell is used to support updates and configurations.
 
+#### fiskaltrust.SignatureCloud
+
+The fiskaltrust.SignatureCloud is a pure online solution. The receipt linking is handled entirely online. The advantage of this solution is that no installation or configuration is required for the client and any platform can use this service.
+
+The fiskaltrust.SignatureCloud has two different versions:
```

Comment:
Should we add information on the support policy here? That we support the current version (default version if you don´t use a version header) and the last minor version. @steininger

Reply:
We will keep it as it is for now and adapt the interface-doc if necessary.